pkg/util/util.go

```diff
@@ -32,3 +47,31 @@ func GenerateSchedulerName(schedulerNames []string) string {
 
 	return defaultSchedulerName
 }
+
+// BindLeaderElectionFlags binds the LeaderElectionConfiguration struct fields to a flagset
+func BindLeaderElectionFlags(l *config.LeaderElectionConfiguration, fs *pflag.FlagSet) {
```

Comment:
How about import `k8s.io/component-base/config/options/leaderelectionconfig.go` directly?

Reply:
> How about import k8s.io/component-base/config/options/leaderelectionconfig.go directly?
en.., I want set default value in function `BindLeaderElectionFlags`; if not, must add a function to set LeaderElectionConfiguration  default values.

Reply:
But actually it's a copy from k8s: )